src/core/qgsuserconfigmanager.h

```diff
@@ -0,0 +1,33 @@
+#ifndef QGSUSERCONFIGMANAGER_H
+#define QGSUSERCONFIGMANAGER_H
+
+#include <qgis.h>
+
+class CORE_EXPORT QgsUserConfig

```

Comment:
Can you put this into its own header file, according to [the coding standards](https://www.qgis.org/en/site/getinvolved/development/qgisdevelopersguide/codingstandards.html#id1)?